provider/kubernetes_test.go

```diff
@@ -499,12 +495,7 @@ func TestGetPassHostHeader(t *testing.T) {
 	expected := &types.Configuration{
 		Backends: map[string]*types.Backend{
 			"foo/bar": {
-				Servers: map[string]types.Server{
-					"1": {
-						URL:    "http://10.0.0.1:801",
-						Weight: 1,
-					},
-				},
+				Servers:        map[string]types.Server{},
```

Comment:
Yea, see this line: https://github.com/containous/traefik/pull/1436/files#diff-4c0e9bf481017297ab14b697157b2675R2079